pages/concepts/bank-funding-source.mdx

```diff
@@ -0,0 +1,149 @@
+| [Dwolla.js](https://developers.dwolla.com/guides/dwolla-js/add-a-bank-account#add-a-bank-using-dwollajs)                      | No                                                      | Bank Account Number, <br /> Routing number | 100%              | Checking, <br /> Savings, <br /> Loan, <br /> General Ledger | Tokenized data, <br /> Optional bank verification with [Microdeposits](https://developers.dwolla.com/guides/micro-deposit-verification#verify-a-bank-account-with-micro-deposits) |
+| [Dwolla.js + IAV](https://developers.dwolla.com/guides/dwolla-js/instant-account-verification#add-a-bank-using-dwollajs--iav) | Yes                                                     | Online banking credentials                 | ~85%              | Checking, <br /> Savings                                     | Flexible CSS styling                                                                                                                                                              |
+| [Dwolla + Plaid integration](https://developers.dwolla.com/guides/plaid#plaid-integration)                                    | Yes                                                     | Online banking credentials                 | ~85%              | Checking, <br /> Savings                                     | [Tokenized integration](https://www.dwolla.com/integrations/plaid/)                                                                                                               |
+| Other Approved Third-party Provider                                                                                           | Yes, if part of third party offering                    | Variable                                   | Variable          | Variable                                                     | Variable                                                                                                                                                                          |
+
+As you decide on the various methods for adding a funding-source, a good thing to keep in mind is that a transfer between two parties requires the sending party to have a verified bank account. Bank account verification prior to sending funds is required by the ACH network.
+
+## Bank Addition + Verification methods
+
+### Add a Bank via the API and verify using Microdeposits
+
+**The use case:**
+I want to add a U.S. bank account by supplying an account and routing number. The bank will be unverified until microdeposits are initiated and verified.
```

Comment:
Could we add hyperlinks around initiated and verified that links off to the API reference docs for these calls?